docs/web3-eth.rst

```diff
@@ -1168,17 +1161,18 @@ signTransaction
 
 .. code-block:: javascript
 
-    web3.eth.signTransaction(transactionObject [, callback])
+    web3.eth.signTransaction(transactionObject [, address,] [, callback])
 
 The method ``signTransaction`` signs a transaction with the private key of the given address.
-This method does only work if you're connected to a Parity node.
+It will sign the transaction locally if a local unlocked account with the given address is existing.
```

Comment:
For L1166-L1167, rephrase to start with the verb describing the return. In context we already know it is ``signTransaction``. Should also include a clarification of what happens when the given address is *not* a local unlocked account.

Signs a transaction with the private key of the given ``from`` address.
If the given address is a local unlocked account, the transaction will be signed locally.